blog/notatki-graficzne/index.md

```diff
@@ -0,0 +1,149 @@
+narysowanych materiałów, a także o ilustratorach spotkań i wydarzeń, tworzących
+wizualne podsumowania, które są świetnymi materiałami edukacyjnymi.
+
+## I co dalej zrobić z tą wiedzą?
+
+![](images/upload_0087.png) Warsztaty ukończyłem, dyplom otrzymałem. Przez
+dłuższy czas nie tykałem flamastra, aż pewnego pięknego dnia stwierdziłem, że
+wypróbuję te umiejętności w codziennej praktyce speca od dokumentacji i autora
+radosnej twórczości po godzinach. Poszerzyłem mój zestaw pisaków, zakupiłem
+[biuwar (moje nowe ulubione słowo ;)](https://sjp.pwn.pl/slowniki/biuwar.html) i
+zacząłem eksperymenty z pisanymi i rysowanymi notatkami. Dla mniej
+zorientowanych, biuwar to duży arkusz papieru z kalendarzem, miejscem na
+odręczne rysunki i listę zadań.
+
+## Jak korzystam z wizualnych notatek
+
+![](images/upload_0088.png) Zarówno w pracy jak i po godzinach tworzę listy
+zadań, mapy tematów i sieć połączeń miedzy nimi. Rysunki i zapiski pomagają mi
+spamiętać rzeczy, które aktualnie biorę na tapet. Zapisuję sobie główne tagi,
+którymi kategoryzuję notatki w formacie markdown, w narzędziu Obsidian, któremu poświęcę
+przyszły artykuł :) Rysunki "pączkują" i z biegiem czasu tworzy się mozaika.
+Jeśli tematy znajdują się blisko siebie, znaczy to, że są w jakiś sposób ze sobą
+związane. Za pomocą kolorów i innych metod mogę podkreślać powiązane tematy. Z
+lubością wykreślam zamknięte sprawy. Kiedy na arkuszu robi się zbyt tłoczno albo
+większość kwestii jest rozwiązanych, wydzieram kartkę i biorę się do zapisywania
+kolejnej. Stare zapiski zachowuję, bo stanowią wizualny zapis stanu rzeczy z
+danego momentu projektu. Czasem do pewnych szczegółów warto wrócić.
+
+## W czym pomagają mi wizualne notatki
+
+Notatki wizualne wspierają mnie w zapamiętywaniu nazw projektów, komponentów,
+dat i powiązań. Tak jak notatka na lodówce, wzrok często spoczywa na arkuszu i
+pomaga mi to odświeżać kluczowe informacje. Im bardziej złożona struktura tego
+nad czym pracujesz, tym bardziej wizualizacja pomaga Ci ułożyć to sobie w
+głowie. Jest to szczególnie pomocne podczas etapu analizy. Dodatkowym plusem
+jest to, że kiedy masz niezapowiedzianą rozmowę na Teams na temat danego
+projektu, masz przed oczami kluczowe fakty. No i oczywiście masz miejsce na
+arkuszu na zrobienie szybkich notatek w trakcie rozmowy. Czasem za status mojego
+projektu wystarcza zdjęcie rysunku podsumowującego moje działania. Potrzebujesz
+zaprojektować diagram? Do dzieła, papier wszystko przyjmie!
+
+## Dlaczego uważam, że warto notować i rysować
+
+![](images/upload_0089.png) Graficzne notatki mogą urozmaicić Twój dzień, jeśli
+czujesz pewien przesyt od patrzenia w monitor i stukania na klawiaturze. Uważam,
+że rysowanie jest rozwijającą czynnością, która angażuje dodatkowe obszary
+mózgu, wspierając kreatywność, wyobraźnie i myślenie. Najlepsze rzeczy w życiu
+są najprostsze. Rysowanie jest niezwykle relaksującą i rozwijającą czynnością.
```

Comment:
Dwie linijki wyżej masz napisane prawie to samo.
```suggestion
są najprostsze.
```

Reply:
ok